aws/rust-runtime/aws-types/src/config.rs

```diff
@@ -60,6 +64,40 @@ impl Builder {
         self
     }
 
+    /// Set the retry_config for the builder
+    ///
+    /// # Examples
+    /// ```rust
+    /// use aws_types::config::Config;
+    /// use smithy_types::retry::RetryConfig;
+    /// let config = Config::builder().retry_config(RetryConfig::new()).build();
```

Comment:
this would just use the default settings right? Maybe we should have the example set the number of retries to 5 or something